### DISCUSSION TITLE
性能优化

### DISCUSSION THREAD

**Discussion by xxx252525** [1Panel-dev/1Panel#5078] [Q&A]
### 1Panel 版本

1.10.7-lts

### 请描述您的需求或者改进建议

1panel面板和docker本身内存占用过高，一些在docker里面的容器占用也很高，我在某云服务器上部署了Halo、PHP、MySQL、openresty服务，网站一运行就占了70%多内存，没安装之前，内存只占用不到3%，安装之后，内存几乎快没了，在Halo编写文章的时候更是由于读写的问题，内存快满了，导致docker服务直接掉了。了，网站直接崩溃了。

### 请描述你建议的实现方案

建议在docker部署应用和1panel本身进行内存占用的优化，减少内存消耗。

**Comment by wanghe-fit2cloud** [comment]
感谢反馈。
1Panel 系统自身占用内存会持续优化，目前应该在 200MB 左右的水平。

> 具体应用的内存占用问题，我们目前还做不到优化，建议反馈应用官方。
